core/src/main/java/io/ap4k/kubernetes/decorator/AddRoleBindingDecorator.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2018 The original authors.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * 
+**/
+package io.ap4k.kubernetes.decorator;
+
+import io.ap4k.Resources;
+import io.ap4k.deps.kubernetes.api.model.KubernetesListBuilder;
+import io.ap4k.deps.kubernetes.api.model.rbac.KubernetesRoleBindingBuilder;
+
+/**
+ * AddRoleBindingDecorator
+ */
+public class AddRoleBindingDecorator extends Decorator<KubernetesListBuilder> {
```

Comment:
How is this relevant to making things work better with Java 11?

Reply:
It won't. I guess I need to rebase.